botocore/config.py

```diff
@@ -219,6 +219,12 @@ class Config:
     :param disable_request_compression: Disables request body compression if
         set to True.
 
+        Defaults to None.
+
+    :type account_id_endpoint_mode: str
+    :param account_id_endpoint_mode: Enables or disables account ID based
+        endpoint routing for supported operations.
+
         Defaults to None.
```

Comment:
We should explain the default behavior in more detail.

Reply:
Yeah I'll add more info for this as well as the available options